datalad/distribution/create_sibling.py

```diff
@@ -287,6 +302,21 @@ def _create_dataset_sibling(
     return remoteds_path
 
 
+def _ls_remote_path(ssh, path):
+    try:
+        # yoh tried ls on mac
+        out, err = ssh("ls -a1 {}".format(sh_quote(path)))
```

Comment:
Why not just `-A`, eliminating the need to filter out `.` and `..` ourselves?

Reply:
good idea! will do